views/score.py

```diff
@@ -264,15 +264,18 @@ def generate_query_string(request):
     return query_string
 
 
-def _retrieve_blank_periods_by_student(persons, periods, scores, reverse=False):
+def _retrieve_blank_periods_by_student(persons, to_exclude, periods, scores, reverse=False):
     # get students with blank periods, students without blank periods if reverse set to true
     students = {person: [] for person in persons}
     students_without_grades = {}
     students_with_grades = {}
     for student, period in scores:
         students[student].append(period)
     for student in students.keys():
```

Comment:
Je viens d'apprendre (à vérifier) que lorsque tu itères sur un dictionnaire; par défaut il le fait sur les clés; donc pas besoin du .keys() apparemment ?